libs/blocks/quiz/quiz.css

```diff
@@ -1,980 +1,384 @@
 
-.dot-indicators:last-of-type .dot.current::before {
-  background-color: var(--color-white);
-  border: 2px solid var(--color-white);
+.dot-indicators.top {
+  display: none;
+  position: fixed;
+  top: 25px;
 }
 
-.dot-indicators:last-of-type .dot.prev::before {
-  background-color: var(--color-white);
+.dot-indicators.bottom{
+  margin-bottom: var(--spacing-s);
 }
 
-.dot-indicators:last-of-type .dot:not(.current):not(.prev)::before {
+.dot-indicators.bottom .dot::before {
   background-color: transparent;
   border: 2px solid var(--color-white);
 }
 
-.dot-indicators:last-of-type .dot::after {
-  display: block;
-  content: ' ';
-  width: calc(100% - 23px);
-  height: 2px;
+.dot-indicators.bottom .dot::after {
   border-bottom: 2px dashed var(--color-white);
-  position: absolute;
-  top: 6px;
-  margin-left: 20px;
 }
 
-@media screen and (min-width: 600px) {
-  .quiz-container .action-area {
-    display: flex;
-    flex-wrap: wrap;
-    align-items: center;
-    gap: var(--spacing-xs);
-  }
-
-  .quiz-container .background .mobileOnly,
-  .quiz-container .background .desktopOnly {
-    display: none;
-  }
-
-  .quiz-container .background .tabletOnly {
-    display: block;
-  }
-
-  /* Split */
-  .quiz-container.split {
-    display: flex;
-    justify-content: center;
-  }
-
-  .quiz-container.split .action-area {
-    gap: var(--spacing-s);
-    flex-direction: row;
-    align-items: center;
-  }
-
-  .quiz-container.split .foreground.container {
-    flex-direction: row;
-    padding: var(--spacing-xl) 0;
-  }
-
-  .quiz-container.split .foreground.container .text {
-    max-width: calc(50% - var(--grid-column-width));
-  }
-
-  .quiz-container.split .media.bleed {
-    position: absolute;
-    width: 50vw;
-    right: 0;
-    height: 100%;
-  }
-
-  .quiz-container.split:not(.row-reversed) .media-credit {
-    text-align: right;
-  }
-
-  .quiz-container.split .text .icon-area img {
-    max-height: 56px;
-    max-width: 212px;
-  }
-
-  .quiz-container.split.large .text .icon-area img {
-    max-height: 80px;
-  }
-
-  .quiz-container.split.row-reversed .media.bleed {
-    right: auto;
-    left: 0;
-  }
-
-  .quiz-container.split.row-reversed .foreground.container {
-    justify-content: flex-end;
-  }
-
-  .quiz-container.split .media.bleed picture,
-  .quiz-container.split .media.bleed video {
-    height: 100%;
-    object-fit: fill;
-  }
-
-  .quiz-container.split.large .text {
-    margin: 0;
-  }
-
-  .quiz-container.split.small .media img,
-  .quiz-container.split.large .media img,
-  .quiz-container.split .media img,
-  .quiz-container.split.small .media video,
-  .quiz-container.split.large .media video,
-  .quiz-container.split .media video {
-    max-height: initial;
-  }
+.dot-indicators.bottom .dot.current::before {
+  background-color: var(--color-white);
+  border: 2px solid var(--color-white);
+}
 
-  .quiz-container.split .button-justified-mobile {
-    width: auto;
-  }
+.dot-indicators.bottom .dot.prev::before {
```

Comment:
Seems to me that the styles are similar to the ones for `.dot-indicators.bottom .dot.current::before`, maybe they can be grouped together